src/Modules/OrchardCore.Commerce.Tax/Views/TaxRateSettings_Edit.cshtml

```diff
@@ -58,9 +60,19 @@
         <td>
             <input :name="'ISite.Rates[' + index + '].TaxCode'" v-model="rate.taxCode">
         </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].VatNumber'" v-model="rate.vatNumber">
+        </td>
         <td>
             <input :name="'ISite.Rates[' + index + '].TaxRate'" v-model="rate.taxRate" type="number" min="0" max="100" step="0.001">
         </td>
+        <td>
+            <select :name="'ISite.Rates[' + index + '].IsCorporation'" v-model="rate.isCorporation">
+                <option :value="IsCorporation.Checked">@T["Checked"]</option>
+                <option :value="IsCorporation.Unchecked">@T["Unchecked"]</option>
+                <option :value="IsCorporation.Ignored">@T["Ignored"]</option>
```

Comment:
I think these values can be inserted using Razor instead of bound via Vue (e.g. `value="@((int)IsCorporation.Ignored)"`), because you do the same with the text anyway so we gain no dynamism from binding the value..